lessons/1.0-Basic-Syntax/text.adoc

```diff
@@ -0,0 +1,13 @@
+= Basic Syntax
+
+Let's start, with a simple example program. The code, you see nearby, is
+printing a number. Let's dissect each line separately.
+
+Line 1 _includes_ the contents of another file. In this case, it is a file,
+called `iostream`, conveniently provided to us by the C++ _standard library_.
+
+Line 4 defines the _entry point_ to our program, from which it starts. The
+entry point *has to be* named `main`.
```

Comment:
> The entry point on *most* platforms is named `main`.

Maybe should mention hosted and freestanding implementations as well?

Reply:
I definitely don't want to talk about that in a lesson targeted at beginners. Maybe drop that sentence altogether?